extensions/identity-hub-client/src/main/java/org/eclipse/dataspaceconnector/identityhub/client/VerifiableCredential.java

```diff
@@ -0,0 +1,67 @@
+/*
+ *  Copyright (c) 2022 Microsoft Corporation
+ *
+ *  This program and the accompanying materials are made available under the
+ *  terms of the Apache License, Version 2.0 which is available at
+ *  https://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  SPDX-License-Identifier: Apache-2.0
+ *
+ *  Contributors:
+ *       Microsoft Corporation - initial API and implementation
+ *
+ */
+
+package org.eclipse.dataspaceconnector.identityhub.client;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.databind.annotation.JsonDeserialize;
+import com.fasterxml.jackson.databind.annotation.JsonPOJOBuilder;
+
+import java.util.Objects;
+
+/**
+ * <p>
+ *   Represents a VerifiableCredential that can be stored in the Identity Hub. The model follows the  <a href="https://www.w3.org/TR/vc-data-model/">W3C Verifiable Credential</a> specification.
+ * </p>
+ * <p>
+ *   The MIME type "application/vc+ldp" should be used to identify the correct format when passing Verifiable Credentials as data to the Identity Hub.
+ * </p>
+ */
+// TODO: implement Verifiable Credential model
+@JsonDeserialize(builder = VerifiableCredential.Builder.class)
+public class VerifiableCredential {
```

Comment:
could we move this class to a `spi` module and avoid duplicating it?

Reply:
Thanks! I moved it to models/credentials/VerifiableCredential.java in the module  `identity-hub-spi`

Reply:
TODO: Add both spi modules in a module called spi like in EDC